bundle/edu.gemini.itc.web/src/main/java/edu/gemini/itc/web/html/PrinterBase.java

```diff
@@ -77,14 +77,13 @@ protected void _printImageLink(final UUID id, final SpcChartType type, final Plo
     }
 
     protected void _printImageLink(final UUID id, final SpcChartType type, final int index, final PlottingDetails pd) {
-        _print("<IMG alt=\"" + toImgAlt(type, index) + "\" height=500 src=\"" + ServerInfo.getServerURL() +
+        _print("<img alt=\"" + toImgAlt(type) + "\" src=\"" + ServerInfo.getServerURL() +
```

Comment:
```width``` and ```height``` are  not mandatory, so I though as long as I just want to display the chart in its original size without any further layout restrictions I can just as well omit them.